trigger/patch.go

```diff
@@ -273,15 +273,15 @@ func (t *patchTriggers) makeData(sub *event.Subscription) (*commonTemplateData,
 			Color:     slackColor,
 		})
 	}
-	var makespan time.Duration
-	if utility.IsZeroTime(t.patch.FinishTime) {
-		patchTasks, err := task.Find(task.ByVersion(t.patch.Id.Hex()))
-		if err == nil {
-			_, makespan = task.GetTimeSpent(patchTasks)
-		}
-	} else {
-		makespan = t.patch.FinishTime.Sub(t.patch.StartTime)
+
+	tasks, err := task.Find(task.ByVersion(t.patch.Id.Hex()))
```

Comment:
I changed it because it wasn't a pattern followed in the whole function and I don't see why this case is different than other cases where we do return on error.